lib/travis/api/v3/services/requests/create.rb

```diff
@@ -13,9 +13,10 @@ def run
 
       user      = find(:user) if access_control.full_access? and params_for? 'user'.freeze
       user    ||= access_control.user
-      remaining = remaining_requests(repository)
+      max       = limit(repository)
+      remaining = remaining_requests(max, repository)
 
-      raise RequestLimitReached, repository: repository if remaining == 0
+      raise RequestLimitReached, repository: repository, max_requests: max, per_seconds: TIME_FRAME.to_i if remaining == 0
```

Comment:
Where is this extra information being used? Are we telling the user somewhere the max requests per hour or per second they can do anywhere?